c2corg_ui/static/js/imageuploader.js

```diff
@@ -477,7 +477,9 @@ app.ImageUploaderModalController.prototype.close = function() {
  * @export
  */
 app.ImageUploaderModalController.prototype.save = function() {
+  $('.img-upl-button').addClass('disabled');
```

Comment:
I think I have copied some code found somewhere else in the UI :P 
How would you recommend to do it?